### PR TITLE
change the readme

### PR DESCRIPTION
**Pull request by Mi3liX9**

### CODE REVIEW

**Review by alialbahrani756 — APPROVED** [review]

**Review by alialbahrani756 — APPROVED** [review]

**Review by Mi3liX9 — COMMENTED** [review]
Good start

**Review by alialbahrani756 — APPROVED** [review]
Hi

**Review by alialbahrani756 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,54 +1,6 @@
-# Astro Starter Kit: Basics
+# Labrary App
```
Comment (alialbahrani756) [inline README.md:1]: Labrary App

**Review by alialbahrani756 — APPROVED** [review]
Hi

**Review by HassanAlabdulal — COMMENTED** [review]
Checked ✅

**Pull request merged**